### ISSUE TITLE
Error when running Monkey Patch on M1 CPU

### ISSUE THREAD

**Issue by effinsky** [iocave/monkey-patch#44]
Encountering an error when running the extension on M1 CPU architecture:

<img width="518" alt="Screenshot 2022-07-06 at 13 55 15" src="https://user-images.githubusercontent.com/54535543/177544841-7b7352eb-713f-4b16-a1e0-b96b1a963a8c.png">

The Mac I'm on: 
<img width="696" alt="Screenshot 2022-07-06 at 13 55 55" src="https://user-images.githubusercontent.com/54535543/177544979-26fdfdd7-df5c-44d6-827b-18a09287e90b.png">

I'll do my best to provide additional info as needed.

Thanks in advance!!

**Comment by knopp** [comment]
I'm not sure what `./applet` is. Monkey patch should run any such executable.

**Comment by effinsky** [comment]
yeah, no idea either. That's just what I got. Any way I could help you shed some light on this to actually use Monkey Patch?

**Comment by knopp** [comment]
https://github.com/microsoft/vscode/issues/136223

Looks like one of the packages inside vscode is intel only and requires Rosetta.

I'm not sure why you don't get the prompt though.

**Comment by effinsky** [comment]
ok, so after installing Rosetta, I get the following error on trying to enable MonkeyPatch. Help getting out of this little rut appreciated :) Maybe it's a permissions issue?

<img width="441" alt="Screenshot 2022-07-09 at 11 08 25" src="https://user-images.githubusercontent.com/54535543/178099471-60de45a8-5d58-41ca-bc57-6c2a557bb5bd.png">

**Comment by alfuken** [comment]
Same thing as above. But doesn't "settle" for me.

```
/var/folders/z3/dfx4v1vd1r952_nn9ld5lvbm0000gn/T/8b9c7930-8add-4650-ac33-d7916987cb30
/script.sh cp: /private/var/folders/z3/dfx4v1vd1r952_nn9ld5lvbm0000gn/T/AppTranslocat
ion/55693252-7D70-4AC1-9212-17BAD3310876/d/Visual Studio Code.app/Contents/Resources/
app/out/bootstrap-amd.js.monkey-patch.backup: Read-only file system cp: /private/var/
folders/z3/dfx4v1vd1r952_nn9ld5lvbm0000gn/T/AppTranslocation/55693252-7D70-4AC1-9212-
17BAD3310876/d/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js: Re
ad-only file system cp: /private/var/folders/z3/dfx4v1vd1r952_nn9ld5lvbm0000gn/T/AppT
ranslocation/55693252-7D70-4AC1-9212-17BAD3310876/d/Visual Studio Code.app/Contents/R
esources/app/out/vs/code/electron-browser/workbench/workbench-monkey-patch.html: Read
-only file system
```

**Comment by knopp** [comment]
You are either running vscode from the ~/Downloads folder or the downloaded flag hasn't been removed from the bundle. See https://github.com/iocave/monkey-patch/issues/6#issuecomment-554706286 .
